dockstore-webservice/src/main/resources/migrations.1.5.0.xml

```diff
@@ -165,4 +165,16 @@
             <column name="dbupdatedate" type="timestamp"/>
         </addColumn>
     </changeSet>
+    <changeSet author="gluu" id="migrate-legacy-verification-information">
+        <sql dbms="postgresql">
```

Comment:
Use the comment attribute to describe these (had to stare at this this figure out what was different between the copy-paste)